workshops/portfolio/README.md

```diff
@@ -61,7 +61,7 @@ And now we're done creating a GitHub account!
 
 Now that we have a GitHub account:
 
-1. Open [jsbin.com][jsbin]
+1. Open [jsbin.com][jsbin] in a new window
```

Comment:
I've seen many people run into the problem of:

1) Clicking on a link
2) Then the new page opens up in the same tab
3) Then they're not sure where the directions went and are unsure of how to go back to the directions